ci/pipelines/deploy-to-test.yml

```diff
@@ -1718,6 +1721,7 @@ jobs:
         trigger: true
       - get: nginx-proxy-ecr-registry-test
         trigger: true
+        passed: [build-and-push-nginx-proxy-to-test-ecr]
```

Comment:
I was wondering how this would affect the [frontend group](https://pay-cd.deploy.payments.service.gov.uk/teams/pay-dev/pipelines/deploy-to-test?group=frontend) but it appears to be [ok](https://pay-cd.deploy.payments.service.gov.uk/teams/pay-dev/pipelines/deploy-to-test/jobs/deploy-frontend/builds/299).